src/homework/HomeWork.kt

```diff
@@ -1,5 +1,33 @@
 package homework
 
+fun incrementNumberByTwo(startingNumber: Int): Int{
+    val incrementedNumber = startingNumber + 2;
+    return incrementedNumber
+}
+
+fun printEvenNumbers(start: Int, finish: Int){
+    var x = start;
+    if (x % 2 == 0) {
+        print("$x");
+        x++;
+        x++;
+        while (x <= finish) {
```

Comment:
Good effort, though I'm confused about the extra x++ and 2 while loops everywhere.
Think about the algorithm.  You seem to have an idea on what needs to get done.
Though if you want to iterate over start to finish, you could have just done:
```for (x in range start..finish)```  And not have to assign x to start, and do x++ anywhere.